### DISCUSSION TITLE
FAQ

### DISCUSSION THREAD

**Discussion by mysterion** [mysterion/aframe-vr-player#4] [Q&A]
### Q: Black screen after selecting video on Google Chrome
~Chrome can play videos with resolution lower or equal to your mobile screen resolution.~
~Example: Playing 4k video(3840 x 2160) on a 2400x1080 device will show a black screen.~
Make sure the video codec and resolution is supported by your phone/chrome, goto `chrome://gpu` -> `Video Acceleration Information` to be sure. thanks @shatsky

### Q: Cant access the links on mobile
Fix 1: make sure the application executable is whitelisted in firewall
![firewall1](https://github.com/user-attachments/assets/7c094a62-eafc-43ff-8042-d5f049d0b36c)

![firewall2](https://github.com/user-attachments/assets/6d297b36-7735-4e07-bd57-52834a00043e)

### Q: `Client sent an HTTP request to an HTTPS server` on mobile device
Make sure you follow these steps:-
1) put `https` explicitly into the browser address bar
![image](https://github.com/user-attachments/assets/d0f960ca-b3b3-415a-8947-0b586bdf04da)
2) tap `advanced`
![image](https://github.com/user-attachments/assets/f9a40880-87fb-4652-854a-3274eba0b46f)
3) tap `proceed to ...`
![image](https://github.com/user-attachments/assets/79fc0fe8-0d07-4c58-b8a3-862b8942e23d)
4) tap `VR` button
![image](https://github.com/user-attachments/assets/3e324675-c11f-487b-8486-901a0896c24d)
5) allow site to access VR device
![image](https://github.com/user-attachments/assets/1d5e9927-5728-4537-a860-4e01655f3a18)
6) [CARDBOARD ONLY] make sure the white line should be in-between the left eye view and right eye view. If its not, put your phone into Landscape mode/orientation, then tap the `VR` button.
![image](https://github.com/user-attachments/assets/18e36ee0-10b5-40d8-82ff-9a695f1f822d)

**Comment by shatsky** [comment]
I believe that "Chrome can play videos with resolution lower or equal to your mobile screen resolution" is incorrect. On Pixel 6 with 2400x1080 display aframe-vr-player on Chrome can certainly play 3840x2160 videos, but attempt to play higher resolution videos results in default background (not black screen). I found that "chrome://gpu" shows supported accelerated video decoding formats with resolution ranges under "Video Acceleration Information", and on Pixel 6 resolution limit is exactly 3840x2160 for most of them (though it can decode 8bit HEVC in 4096x2176). However if opening video file directly, Chrome is able to play even 8k (though not without freezes). I guess there is a bug or shortcoming in underlying pipeline which prevents using software-decoded video frame as texture.

Testing note: to open local file in Chrome on Android directly, copy local "file://" URL to clipboard, activate Chrome URL input and choose "Text you copied" action (if you enter such URL into the input, Chrome will only provide options to search for it via search engine, possibly another bug in current Chrome).

**Reply by mysterion** [reply]
tested it a few months ago on few 4k videos, must be an 'unsupported codec issue' and not resolution problem. i'll updated  the faq, thanks.
